pkg/urbit/worker/mars.c

```diff
@@ -287,6 +289,7 @@ _mars_sure_feck(u3_mars* mar_u, c3_w pre_w, u3_noun vir)
 
   mar_u->rec_o = c3o(mar_u->rec_o, rec_o);
   mar_u->pac_o = c3o(mar_u->pac_o, pac_o);
+  mar_u->mel_o = c3a(u3_Host.ops_u.mel, c3o(mar_u->mel_o, mel_o));
```

Comment:
`u3_Host.ops_u` isn't propagated from urth to mars. This "works" because a loobean "bunts" to `c3y`, given zero-initialization.

To be correctly propagated, `--auto-meld` should become a bit in `u3C.wag_w`.

Reply:
This needs to be something like `(u3C.wag_w & u3o_auto_meld) ? c3n : c3o(mar_u->mel_o, mel_o)`